umh-core/pkg/fsmv2/benthos_monitor/throughput_red_test.go

```
@@ -0,0 +1,266 @@
+	var outputCounter int64 = 5
+
+	srv := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		switch r.URL.Path {
+		case "/ping":
+			_, _ = w.Write([]byte("pong"))
+		case "/ready":
+			_, _ = w.Write([]byte(`{"error":""}`))
+		case "/version":
+			_, _ = w.Write([]byte(`{"version":"1.2.3"}`))
+		case "/metrics":
+			fmt.Fprintf(w, "input_received{label=\"\",path=\"root.input\"} %d\n", inputCounter)
```

golangci-lint (2.12.2)

[error] Error return value of `fmt.Fprintf` is not checked

(errcheck)